code/modules/mob/living/carbon/carbon.dm

```diff
@@ -615,10 +615,12 @@ var/list/ventcrawl_machinery = list(/obj/machinery/atmospherics/unary/vent_pump,
 	return TRUE
 
 /mob/living/carbon/restrained()
-	if(handcuffed)
+	if(get_retraining_item())
 		return TRUE
-	return
+	return FALSE
```

Comment:
could just do return `get_retraining_item()` and no if statements

Reply:
I prefer to let the procs return what they say they will return. restrained asks the question if the mob is restrained or not so I will suspect a bool to be returned